css/properties/background-image.json

```diff
@@ -176,12 +176,16 @@
               "edge": "mirror",
               "firefox": [
                 {
-                  "version_added": "88",
-                  "notes": "Before Firefox 89, the <code>type()</code> function is not supported as an argument to <code>image-set()</code>."
+                  "version_added": "89"
                 },
                 {
                   "prefix": "-webkit-",
                   "version_added": "90"
+                },
+                {
+                  "version_added": "88",
+                  "version_removed": "89",
+                  "notes": "The <code>type()</code> function is not supported as an argument to <code>image-set()</code>."
```

Comment:
I'm a little confused here. I think the entries like this were intended to be `partial_implementation`? A long time ago we used separate support statement for notes like this, but I didn't think that was the convention of late.

Reply:
Yes, let's do that.

Reply:
I even had "Use `partial_implementation` for that single-version issue" in the description but didn't do it...